File: msg-sentiment/src/test/java/dev/number6/sentiment/adaptor/FakeDatabaseAdaptor.java
package dev.number6.sentiment.adaptor;

import dev.number6.comprehend.results.PresentableEntityResults;
import dev.number6.comprehend.results.PresentableKeyPhrasesResults;
import dev.number6.comprehend.results.PresentableSentimentResults;
import dev.number6.db.port.DatabasePort;

import java.time.LocalDate;
import java.util.Collection;

public class FakeDatabaseAdaptor implements DatabasePort {
    @Override
    public void createNewSummaryForChannels(Collection<String> channelNames, LocalDate comprehensionDate) {

    }

    @Override
    public void save(PresentableSentimentResults results) {

    }

    @Override
    public void save(PresentableEntityResults results) {

    }

    @Override
    public void save(PresentableKeyPhrasesResults results) {

    }
}

File: msg-entity/src/test/java/dev/number6/entity/adaptor/FakeDatabaseAdaptor.kt
package dev.number6.entity.adaptor

import dev.number6.comprehend.results.PresentableEntityResults
import dev.number6.comprehend.results.PresentableKeyPhrasesResults
import dev.number6.comprehend.results.PresentableSentimentResults
import dev.number6.db.port.DatabasePort
import java.time.LocalDate

class FakeDatabaseAdaptor : DatabasePort {
    override fun createNewSummaryForChannels(channelNames: Collection<String>, comprehensionDate: LocalDate) {}
    override fun save(results: PresentableSentimentResults) {}
    override fun save(results: PresentableEntityResults) {}
    override fun save(results: PresentableKeyPhrasesResults) {}
}